test/e2e/testobjects.go

```diff
@@ -52,6 +52,43 @@ const (
              throw err
           }
 `
+
+	javabuilder = `
+          try {
+             timeout(time: 20, unit: 'MINUTES') {
+        
+                node("java-builder") {
+                    container("java") {
+                        sh "mvn --version"
+                        
+                    }
+                }
+             }
+          } catch (err) {
+             echo "in catch block"
+             echo "Caught: ${err}"
+             currentBuild.result = 'FAILURE'
+             throw err
+          }
```

Comment:
Any particular reason why this is indented this far? Typically I don't see indents for golang string literals

Reply:
the jenkins console I ran this test from to confirm function indented it and copied form there directly

I can try to manually remove indents if need be